symbolic-debuginfo/src/function_builder.rs

```diff
@@ -17,8 +19,13 @@ pub struct FunctionBuilder<'s> {
     address: u64,
     /// The size of the outer function.
     size: u64,
-    /// The inlinees, in any order. They will be sorted in `finish()`.
-    inlinees: Vec<FunctionBuilderInlinee<'s>>,
+    /// All inlinees at any depth inside this function.
+    ///
+    /// These are stored in a `BinaryHeap<Reverse<_>>` so that `.pop()` returns them ordered by
+    /// address, from low to high. (A `BinaryHeap` by itself is a max-heap, so we use `Reversed` to
+    /// make this a min-heap). We use a heap instead of a sorted `Vec` because we may need to insert
```

Comment:
```suggestion
    /// address, from low to high. (A [`BinaryHeap`] by itself is a max-heap, so we use [`Reverse`] to
    /// make this a min-heap). We use a heap instead of a sorted `Vec` because we may need to insert
```

TIL: https://doc.rust-lang.org/std/cmp/struct.Reverse.html